application/static_site/views.py

```diff
@@ -41,14 +41,19 @@ def ethnicity_in_the_uk():
 @static_site_blueprint.route("/ethnicity-in-the-uk/<page_name>")
 @login_required
 def ethnicity_in_the_uk_page(page_name):
-    ETHNICITY_IN_THE_UK_PAGES = ["ethnic-groups-and-data-collected", "ethnic-groups-by-sexual-identity"]
-    if page_name in ETHNICITY_IN_THE_UK_PAGES:
+    if page_name.lower() in {"ethnic-groups-by-sexual-identity"}:
```

Comment:
Hardcode to `if page_name == "ethnic-groups-by-sexual-identity"`?

Reply:
Yeah I probably should have done that, but it doesn't feel worth changing tbh.